tests/rptest/tests/group_membership_test.py

```diff
@@ -113,3 +118,163 @@ def _check_no_duplicates():
                    10,
                    backoff_sec=2,
                    err_msg="found persistent duplicates in groups listing")
+
+
+class GroupMetricsTest(EndToEndTest):
```

Comment:
I think your test would be a lot simpler if you based it on `RedpandaTest` rather than `EndToEndTest`. The former takes care of things like starting the cluster, creating test topics, etc...